lib/transport/tls-context.c

```diff
@@ -346,6 +347,9 @@ tls_context_setup_cmd_context(TLSContext *self)
 
   SSL_CONF_CTX_free(ssl_conf_ctx);
   return result;
+#else
```

Comment:
This patch is getting there, I'd have one last improvement suggestion here: we should return FALSE in case conf_cmds_list is non-NULL.

The code today does not allow anything else as tls_context_set_conf_cmds() fails, but just in case something else starts to use the same mechanism, I'd make sure that we fail here.

Reply:
OK, done